cmd/kubeadm/app/phases/kubelet/kubelet.go

```diff
@@ -55,9 +58,17 @@ func CreateBaseKubeletConfiguration(cfg *kubeadmapi.MasterConfiguration, client
 		return err
 	}
 
+	var majorMinorVersion string
+	version := strings.Split(strings.TrimPrefix(cfg.KubernetesVersion, "v"), ".")
```

Comment:
I'd use `pkg/util/version.ParseSemantic` here and then use `fmt.Sprintf("%s.%s",  v.Major(), v.Minor())`